src/python/artm_vs2012.sln

```diff
@@ -1,62 +0,0 @@
-﻿
```

Comment:
I don't see any reason to remove VS project. I prefer VS + pylint and dislike PyCharm. So, if it's not critical, I'd like the project file to present in a master branch.